shoop/testing/management/commands/shoop_populate_mock.py

```diff
@@ -26,7 +26,8 @@ def handle(self, *args, **options):
         translation.activate(settings.LANGUAGES[0][0])
         superuser_name = options.get("with_superuser")
         if superuser_name:
-            user = get_user_model().objects.filter(username=superuser_name).first()
+            filter_params = {get_user_model().USERNAME_FIELD: superuser_name}
+            user = get_user_model().objects.filter(**filter_params).first()
             if not user:
                 user = get_user_model().objects.create_superuser(
                     username=superuser_name,
```

Comment:
Yes, it works as soon as the `create_superuser` function has a `kwargs` parameter. In this case, the developer must keep in mind that any hardcoded keyword can be used when invoking this method.
If the custom user model `USERNAME_FIELD` is `email`, for example, the `create_superuser` here will fail, because you are setting the same parameter twice (1 harcoded and 1 through `USERNAME_FIELD`).